README.md

```diff
@@ -7,56 +7,55 @@
-    - [Releasing for FreeBSD](#releasing-for-freebsd)
-    - [Releasing for Windows](#releasing-for-windows)
-    - [Releasing for MacOS](#releasing-for-macos)
-  - [Utilities](#utilities)
-    - [Checking Out Source](#checking-out-source)
-    - [Cross-Platform Builds](#cross-platform-builds)
-    - [Sanity Checking the Bundle](#sanity-checking-the-bundle)
-    - [Auto-Generating Manifests](#auto-generating-manifests)
-  - [Deploying Infrastructure](#deploying-infrastructure)
+   - [Release labels](#release-labels)
+- [Onboarding a New Plugin](#onboarding-a-new-plugin)
+- [Building and Testing an OpenSearch Distribution](#building-and-testing-an-opensearch-distribution)
+  - [Building from Source](#building-from-source)
+  - [Assembling a Distribution](#assembling-a-distribution)
+  - [Building Patches](#building-patches)
+  - [Min snapshots](#min-snapshots)
+  - [CI/CD Environment](#cicd-environment)
+  - [Build Numbers](#build-numbers)
+  - [Latest Distribution Url](#latest-distribution-url)
+  - [Testing the Distribution](#testing-the-distribution)
+  - [Checking Release Notes](#checking-release-notes)
+  - [Signing Artifacts](#signing-artifacts)
+    - [PGP](#pgp)
+    - [Windows](#windows)
+  - [Signing RPM artifacts](#signing-rpm-artifacts)
+- [Making a Release](#making-a-release)
+  - [Releasing for Linux](#releasing-for-linux)
+  - [Releasing for FreeBSD](#releasing-for-freebsd)
+  - [Releasing for Windows](#releasing-for-windows)
+  - [Releasing for MacOS](#releasing-for-macos)
+- [Utilities](#utilities)
+  - [Checking Out Source](#checking-out-source)
+  - [Cross-Platform Builds](#cross-platform-builds)
+  - [Sanity Checking the Bundle](#sanity-checking-the-bundle)
+  - [Auto-Generating Manifests](#auto-generating-manifests)
+- [Deploying Infrastructure](#deploying-infrastructure)
 - [Contributing](#contributing)
 - [Getting Help](#getting-help)
 - [Code of Conduct](#code-of-conduct)
 - [Security](#security)
 - [License](#license)
 - [Copyright](#copyright)
 
-## Releasing OpenSearch
+### Releasing OpenSearch
 
-### Releases and Versions
+See [release process](./RELEASE_PROCESS_OPENSEARCH.md) document for more information.
+
+#### Releases and Versions
 
 The OpenSearch project releases as versioned distributions of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. It [follows semantic versioning](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/). Software, such as Data Prepper, clients, and the Logstash output plugin, are versioned independently of the OpenSearch Project. They also may have independent releases from the main project distributions. The OpenSearch Project may also release software under alpha, beta, release candidate, and generally available labels. The definition of when to use these labels is derived from [the Wikipedia page on Software release lifecycle](https://en.wikipedia.org/wiki/Software_release_life_cycle). Below is the definition of when to use each label.
```

Comment:
This might confused people thinking dataprepper and clients are using the same way of release procedures.
I think we do not need to mention these specifically.

Reply:
Thanks, moved this section from this file.